framework/src/st_things/things_stack/framework/things_req_handler.c

```diff
@@ -197,17 +197,12 @@ static OCEntityHandlerResult convert_ap_infor(things_resource_s *target_resource
 	} else {
 		rep = target_resource->rep;
 	}
+
+	THINGS_LOG_D(TAG, "Create response for accesspoints resource, WiFi list count is: %d", list_cnt);
```

Comment:
i think this log is not need.
because in file of things_req_handler.c[Line 236], already printed `THINGS_LOG_V(TAG, "Get Access points list!!! %d ", list_cnt);` message.
Please check duplicate message.

Reply:
It is different message.
It(201) will be printed  when received get request about accesspoints resource. and 236 will print when ap list scanning is done.  both messges need for debugging.

Reply:
I don't think so.
this logic is part of `convert_ap_infor()` function.
`convert_ap_infor()` function is called by `get_ap_scan_result()[line 237]` with 3rd parameter only.
So, it's always the same result. 
Please check it.